core/templates/dev/head/pages/collection_editor/editor_tab/collection_node_creator_directive.html

```diff
@@ -5,6 +5,6 @@
 </form>
 or
 <form ng-submit="addExploration()">
-  <input ng-model="newExplorationId" type="text" placeholder="New exploration ID">
-  <button type="submit" ng-disabled="!newExplorationId">Add Existing Exploration</button>
+  <input class="form-inline" ng-model="newExplorationId" type="text" placeholder="New exploration ID">
```

Comment:
This has no effect. And, your branch seems to have the collection editor bug (PR #3410). To access it you have to edit the files locally from the mentioned PR or update if it's merged. 

> I got 401's trying to access the collection editor on my dev server, but the changes seemed trivial enough.

From the development perspective, it is very dangerous to push the code blindly. Please verify the changes locally before making a PR